src/services/templates/joiErrorTemplate.js

```diff
@@ -0,0 +1,52 @@
+import Util from '../../helpers/utils';
+
+const util = new Util();
+export const joiValidationError = (req, res, next, error) => {
+  if (error) {
+    if (
+      error.details[0].message
+        .replace('/', '')
+        .replace(/"/g, '')
+        .includes('email')
+    ) {
+      const Error = {
+        error: error.details[0].message.replace('/', '').replace(/"/g, ''),
+        example: 'xxx@yyy.zzz',
+      };
+      util.setError(400, Error);
+      return util.send(res);
+    }
+    if (
+      error.details[0].message
+        .replace('/', '')
+        .replace(/"/g, '')
+        .includes('fails to match the required')
```

Comment:
templates folder should only contain html code snippets. Put this code in the helpers folder